uxarray/remap/nearest_neighbor.py

```diff
@@ -32,16 +32,16 @@ def _nearest_neighbor(source_grid: Grid,
         Data variable to remaps
     remap_to : str, default="nodes"
         Location of where to map data, either "nodes" or "face centers"
+    coord_type: str, default="latlon"
+        Coordinate type to use for nearest neighbor query, either "latlon" or "cartesian"
```

Comment:
Moving forward, let's refer to lat/lon coords as "spherical", can you make the change throughout to address this?